cartographer_ros/cartographer_ros/node.cc

```diff
@@ -131,6 +136,9 @@ Node::Node(const NodeOptions& node_options, tf2_ros::Buffer* const tf_buffer)
   wall_timers_.push_back(node_handle_.createWallTimer(
       ::ros::WallDuration(node_options_.pose_publish_period_sec),
       &Node::PublishTrajectoryStates, this));
+  wall_timers_.push_back(node_handle_.createWallTimer(
+      ::ros::WallDuration(node_options_.submap_publish_period_sec),
```

Comment:
Done (90bcac6). I have used the same value for publishing submaps (0.3 seconds).

By the way, will this break people's config files? There is no default value provided when loading from the .lua dictionary, i.e. we'll run into trouble if `submap_publish_period_sec` is missing from a config file. I guess this is going to be a bit annoying.